buildchow/templates/defaults.yaml

```diff
@@ -0,0 +1,37 @@
+- defaults:
+    name: global
+    disabled: false
```

Comment:
this just seems completely counter intuitive to me to have jobs as disabled as global default.
That should be set pr. stream, not globally imo.